agi_hoheitsgrenzen_pub_sogis/agi_hoheitsgrenzen_pub_hoheitsgrenzen_bezirksgrenze.sql

```diff
@@ -0,0 +1,14 @@
+SELECT 
+    hoheitsgrenzen_gemeindegrenze.bezirksname,
+    bezirksnummer,
```

Comment:
Schön wäre, wenn auch dieses Attribut mit tabellenname.attributname referenziert wäre. Auch in Zeile 5 und 12